cms/envs/yaml_config.py

```diff
@@ -65,6 +65,34 @@ def construct_yaml_str(self, node):
 SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
 DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
 
+GIT_REPO_EXPORT_DIR = '/edx/var/edxapp/export_course_repos'
+SESSION_INACTIVITY_TIMEOUT_IN_SECONDS = None
+EMAIL_FILE_PATH = None
+STATIC_URL_BASE = None
+STATIC_ROOT_BASE = None
+SESSION_COOKIE_NAME = None
+ADDL_INSTALLED_APPS = []
+AUTH_USE_CAS = False
+CAS_ATTRIBUTE_CALLBACK = None
+MICROSITE_ROOT_DIR = ''
+CMS_SEGMENT_KEY = None
+DATADOG = {}
+ADDL_INSTALLED_APPS = []
+LOCAL_LOGLEVEL = 'INFO'
+##############################################################
+#
+# ENV TOKEN IMPORT
+#
+# Currently non-secure and secure settings are managed
+# in two yaml files. This section imports the non-secure
+# settings and modifies them in code if necessary.
+#
+
+with open(CONFIG_ROOT / CONFIG_PREFIX + "env.yaml") as env_file:
+    ENV_TOKENS = yaml.safe_load(env_file)
+
+ENV_TOKENS = convert_tokens(ENV_TOKENS)
+
```

Comment:
This top block was moved to be above the celery var declarations, as they use `ENV_TOKENS` now.